pkg/resources/helpers_test.go

```diff
@@ -120,6 +120,14 @@ func storageIntegration(t *testing.T, id string, params map[string]interface{})
 	return d
 }
 
+func stream(t *testing.T, id string, params map[string]interface{}) *schema.ResourceData {
```

Comment:
**[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>stream redeclared in this block